books/free-programming-books-subjects.md

```diff
@@ -288,6 +288,7 @@ Books that cover a specific programming language can be found in the [BY PROGRAM
 
 ### Database
 
+* [Comprehensive-DBMS-Guide] (https://annamalaiuniversity.ac.in/studport/download/engg/CSE_Engg/resources/CSE_CSPC403_DBMS.pdf) - Annamalai University DBMS Course PDF (PDF)
```

Comment:
you have not indicated any reason this is valuable. To me, it looks like garbles class notes, definitely not a book or a cheat sheet. Every statement on the first page is wrong. `Annamalai University DBMS Course PDF` is not an author.

Reply:
I apologize for the mistake.
Keeping the bias aside, thanks for the criticism. Will improve.

Regards,